src/preset.ts

```diff
@@ -1949,14 +1954,15 @@ async function expandWorkflowPresetImpl(folder: string, name: string, workspaceF
                 }
             ]
         };
-        return expandWorkflowPresetHelper(folder, preset, workspaceFolder, sourceDir, true);
+        return expandWorkflowPresetHelper(folder, preset, workspaceFolder, sourceDir, true, enableTryApplyDevEnv, errorHandler);
     }
 
     log.error(localize('workflow.preset.not.found', 'Could not find workflow preset with name {0}', name));
+    errorHandler?.tempErrorList.push([localize('workflow.preset.not.found', 'Could not find workflow preset'), name]);
     return null;
 }
 
-async function expandWorkflowPresetHelper(folder: string, preset: WorkflowPreset, workspaceFolder: string, sourceDir: string, allowUserPreset: boolean = false) {
+async function expandWorkflowPresetHelper(folder: string, preset: WorkflowPreset, workspaceFolder: string, sourceDir: string, allowUserPreset: boolean = false, enableTryApplyDevEnv: boolean = true, errorHandler?: ExpansionErrorHandler) {
```

Comment:
Not for this PR, but it might be wise to make a TODO for the fact that now that we have a collection for the Problems Pane for CMakePresets validation and expansion, there are lots of other entry points where we could use it, for example, confirming that step 1 of the workflow preset is a configure step, doesn't need a TODO comment, but maybe we could make a note or issue for ourselves somewhere

Reply:
a work item maybe? it'll also need to be fleshed out a bit so we can add different kinds of errors without wiping the rest

Reply:
Yeah I think that's a good idea